pkg/networkstream/v1/wire.go

```
@@ -0,0 +1,431 @@
+		cmdline = cmdline[:maxCmdlineBytes]
+	}
+
+	total := processNodeOverheadBytes + len(cmdlineTruncationMarker) +
+		escapedLen(cmdline) +
+		escapedLen(node.Comm) + escapedLen(node.Pcomm) + escapedLen(node.Path) +
+		escapedLen(node.Cwd) + escapedLen(node.Hardlink) +
+		escapedLen(node.UserName) + escapedLen(node.GroupName)
+	for _, child := range node.ChildrenMap {
+		total += estimateProcessBytes(child, budget-1)
+	}
+	for i := range node.Children {
```

golangci-lint (2.12.2)

[error] SA1019: node.Children is deprecated: Use ChildrenMap instead 

(staticcheck)